python/tests/compat/test_mantid.py

```diff
@@ -40,6 +40,7 @@ def setUpClass(cls):
         cls.base_event_ws = mantid.LoadEventNexus(
             MantidDataHelper.find_file(filename),
             OutputWorkspace="test_ws{}".format(__file__),
+            SpectrumMax=200,
```

Comment:
Have you verified that these spectra contain actual events? I frequently see workspaces where the first (and last) spectra are empty.

Reply:
We still have 100500 events in the workspace